docs/root/development/performance/device_conditions.rst

```diff
@@ -0,0 +1,73 @@
+.. _device_conditions:
```

Comment:
nit: this is different than the convention we were using elsewhere.

Reply:
Is it?! Let me fix it!